### DISCUSSION TITLE
Piper local neural network TTS engine

### DISCUSSION THREAD

**Discussion by shuvashish76** [koreader/koreader#12517] [Ideas]
https://calibre-ebook.com/whats-new

v7.18 of Calibre added A new Text-to-Speech engine ([Piper](https://github.com/rhasspy/piper)) that uses a neural network for realistic sounding voices which run locally, no cloud services are used.

Is it possible to add similar feature for KOReader?

**Comment by benoit-pierre** [comment]
See #545, #7975, #9548.
